test_site/html/home_with_excluded_elements.htm

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Home Page With Excluded Elements</title>
+    <script type="text/javascript">
+    function removeElement(node) {
```

Comment:
I think this script and some of the html content below is unnecessary.  The next developer in here may not be aware of what is and isn't actually used, so it's best to keep it minimal.

Reply:
Yeah, this is kept from the `home.html` as something to enable an element that isn't immediately present but becomes present later. Which...isn't _strictly_ necessary for the purpose of this test so I'll pare down for sure.